backend/server/dissectors/dissected_request.go

```diff
@@ -117,7 +116,11 @@ func (m *DissectedRequest) FromURL(key string) *Coercable {
 	if m.Error != nil {
 		return makeErrCoercable(m)
 	}
-	value, found := m.urlValues[key]
```

Comment:
The biggest change is that gorilla Mux offers mux.Vars() (from which we created `urlValues`) which gives you all of the url keys and their values. Unfortunately, the closest `chi` comes is that you can use `chi.URLParam(m.Request, key)` to get a specific url param key. 

Because of this, I removed the urlParameters, as we can't use them anymore. I tried to honor the setup we had before, but manually creating that urlParam map with the chi functionality created a TON of boilerplate code for each request (it added two lines per each request, so hundreds of lines), so I just went with adding in this chi functionality here instead.

Reply:
So, it looks like you  _can_ get all of the url parameters, but as a list of keys and a list of values (I don't know why it's not a list of keys _and_ values,  but I don't make the calls... at least here).

If we do want to go down that route, we'd call chi.RouteContext(r.Context()).URLParams

But, since we need this as a map, we either need to revise the dissectors to take the above type, or we need to create some code to re-wrap the values as `map[string]string`

Reply:
ooh good find - yeah weird that it's two lists. Yeah I might just create some code to re-wrap the values, which would allow us to stick closer to the original code